doc/resources/jenkins/Jenkinsfile-webapp-tomcat

```diff
@@ -0,0 +1,18 @@
+node {
+  stage('init') {
+    checkout scm
+  }
+
+  stage('build') {
+    sh '''
+        mvn clean package
+        mv target/todo-app-java-on-azure-1.0-SNAPSHOT.war ROOT.war
```

Comment:
`mv target/*.war ROOT.war`

Reply:
And I think there should be some modification to get the  `,war`, otherwise only `.jar` available if building with todo-app diirectly

Reply:
Add instruction to change the app to use war package